desktop/env.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  isDev: process.env.NODE_ENV === 'development' || process.defaultApp || /node_modules[\\/]electron[\\/]/.test(process.execPath)
+}
```

Comment:
I'm with you but believe it wouldn't be easily possible with our current setup as we don't use webpack for the desktop files.

In case of using webpack we should also think of a solution for building _shared_ modules that can be imported in the `main` as well as the `render` process without adding every module in both bundles. 
From what I know, [electron-webpack](https://github.com/electron-userland/electron-webpack) is doing something like this with the `common` directory.

Checking things like`process.defaultApp` wouldn't be possible in webpack as it's just available in electron – this however shouldn't be a biggy as we can still decide to only rely on `NODE_ENV`.

Reply:
I guess what I'm most curious about is whether this impacts the browser. this app needs to continue working in both environments ; how is not as important 😉

Reply:
The web app and the code base for the electron app don‘t have any overlap atm.